frontend/compositions/dashboard-header/desktop-nav.tsx

```diff
@@ -1,29 +1,32 @@
 import * as React from "react"
 import styles from "./dashboard-header.module.css"
+import Link from "next/link"
+import { useRouter } from "next/router"
 
 export default function DesktopNav() {
-  const [selected, setSelected] = React.useState("Search")
+  const router = useRouter()
 
-  const handleNavChange = (e: React.MouseEvent<Element>): void => {
-    const value = (e.target as HTMLElement).innerText
-    setSelected(value)
-  }
+  const menu = [
+    { title: "Search", path: "/dashboard" },
+    { title: "Profile", path: "dashboard/profile" },
+    { title: "Sign Out", path: "#" }
```

Comment:
We can create a /logout page that handles logging out and then redirects to the landing page. so here let's use `path: "/logout"`.